TIG/TIG/Feature/Home/Timeline/TimelineView.swift

```diff
@@ -68,20 +77,22 @@ fileprivate struct TimelineHeaderView: View {
 fileprivate struct TimelineBodyView: View {
     
     @Bindable private var homeViewModel: HomeViewModel
```

Comment:
바인딩을 사용할 일이 없다면 굳이 @Bindable로 받지 않아도 됩니다.
```suggestion
    @private var homeViewModel: HomeViewModel
```

Reply:
수정했습니다!